Libraries/Components/Button.windows.js

```diff
@@ -155,35 +155,46 @@ class Button extends React.Component {
     if (Platform.OS === "windows") {
       const tabIndex = this.props.tabIndex || 0;
       const windowsTabFocusable = !disabled && tabIndex >= 0;
-      content =
-        <FocusableWindows
-          ref={this._setFocusableRef}
+      return (
+        <Touchable
+          importantForAccessibility="no"
+          testID={testID}
           disabled={disabled}
-          isTabStop={windowsTabFocusable}
-          tabIndex={tabIndex}
-          disableSystemFocusVisuals={this.props.disableSystemFocusVisuals}
-          handledKeyDownKeys={DOWN_KEYCODES}
-          handledKeyUpKeys={UP_KEYCODES}
-          onKeyDown={this._onKeyDown}
-          onKeyUp={this._onKeyUp}
-          onFocus={this._onFocus}
-          onBlur={this._onBlur}
-        >
-          {content}
-      </FocusableWindows>;
+          onPress={onPress}>
+            <FocusableWindows
+              ref={this._setFocusableRef}
+              disabled={disabled}
+              isTabStop={windowsTabFocusable}
+              tabIndex={tabIndex}
+              disableSystemFocusVisuals={this.props.disableSystemFocusVisuals}
+              accessibilityComponentType="button"
+              accessibilityLabel={accessibilityLabel}
+              accessibilityTraits={accessibilityTraits}
+              handledKeyDownKeys={DOWN_KEYCODES}
+              handledKeyUpKeys={UP_KEYCODES}
+              onKeyDown={this._onKeyDown}
+              onKeyUp={this._onKeyUp}
+              onFocus={this._onFocus}
+              onBlur={this._onBlur}
+              onAccessibilityTap = {this._onAccessibilityTap}
```

Comment:
> =  [](start = 32, length = 3)

remove the spaces around `=` to be consistent with the rest of the file

Reply:
done